pkg/db/db_client/db_client.go

```diff
@@ -157,13 +164,12 @@ func (c *DbClient) ServerSettings() *db_common.ServerSettings {
 
 // Close implements Client
 // closes the connection to the database and shuts down the backend
-func (c *DbClient) Close(context.Context) error {
+func (c *DbClient) Close(ctx context.Context) error {
```

Comment:
remove ctx

Reply:
We can't remove this context since the local implementation of DBClient uses the context for service shutdown

Reply:
remove the unused var name then